src/psyclone/domain/gocean/transformations/gocean_const_loop_bounds_trans.py

```diff
@@ -104,25 +108,81 @@ def validate(self, node, options=None):
         :param node: the GOInvokeSchedule to transform.
         :type node: :py:class:`psyclone.gocean1p0.GOInvokeSchedule`
         :param options: a dictionary with options for transformations.
-        :type options: dictionary of string:values or None
+        :type options: dict of str:values or None
 
         :raises TransformationError: if the supplied node is not a \
-                                     GOInvokeSchedule.
+            GOInvokeSchedule.
+        :raises TransformationError: if the supplied schedule has loops with \
+            index offsets
```

Comment:
unsupported index offsets or .... because there are several other checks now (thanks).

Reply:
Oops, extended the raises list now.